private/buf/bufworkspace/testdata/basic/workspacev2_duplicate_path/buf.yaml

```diff
@@ -0,0 +1,18 @@
+version: v2
+modules:
+  - path: ./proto/shared
+    excludes:
+      - proto/shared/prefix/foo
+  - path: proto//shared1
+    includes:
+      - ./proto//shared1/prefix/x/
```

Comment:
Why do these test files have spurious `/`s at the end? This would imply to me (without further comments) that the spurious slashes are what we're intending to test here. It seems like this is a false flag.

Reply:
Removed in [9aaeb3b](https://github.com/bufbuild/buf/pull/3260/commits/9aaeb3be49cc95eae929ff8eaa318b94aeeb8a12) and [613b27b](https://github.com/bufbuild/buf/pull/3260/commits/613b27b6b3cd5da3a0a26c4fe45b1564e6c4be5c)